libvmaf/src/feature/cambi.c

```diff
@@ -1080,7 +1081,8 @@ static double combine_dist_src_scores(double dist_score, double src_score) {
 static int extract(VmafFeatureExtractor *fex,
                    VmafPicture *ref_pic, VmafPicture *ref_pic_90,
                    VmafPicture *dist_pic, VmafPicture *dist_pic_90,
-                   unsigned index, VmafFeatureCollector *feature_collector) {
+                   unsigned index, VmafFeatureCollector *feature_collector,
+				   VmafFrameSyncContext *framesync) {
```

Comment:
Please add the `VmafFrameSyncContext` inside the `VmafFeatureExtractor` struct. That way we do not need to break the `extract` API and touch all these other files.

Reply:
`.extract()` API has been clean up and `framesync` context has been passed inside the `VmafFeatureExtractor` struct.